netfilter_openvpn.py

```diff
@@ -78,6 +78,10 @@
 if hasattr(config, 'LDAP_MEMBER_ATTR'):
```

Comment:
let's reindent this whole file to 4-space tabs, seems like the formatting is a bit strange

Reply:
will their logging work at all? (the mozdef stuff)

Reply:
nope, that's why I disabled it in config and remove from dependencies, but I do not want clean it out now

Reply:
What win will we get from re indenting file(tab -> 4 spaces) besides huge commit?

Reply:
if we're going to use this I'd like to see logging in place so we have an idea of what's going on; we can skip reindenting for now

Reply:
`USE_SYSLOG = True` mean we will get all log messages in openvpn.log (already checked)